spec/plsql/package_spec.rb

```diff
@@ -55,6 +55,11 @@
     expect(plsql.test_package.procedure_defined?(:inexistent_procedure)).to be_falsey
   end
 
+  it "should search objects via []" do
+    expect(PLSQL::Package.find(plsql, :test_package)['test_procedure']).to be_a PLSQL::Procedure
+    expect(PLSQL::Package.find(plsql, :test_package)['test_variable']).to be_a PLSQL::Variable
```

Comment:
I would add test with referencing by symbol to document the capability
I would add test with upcase string if the code will be changed to work with upcase references too.